spark/src/main/java/org/apache/iceberg/actions/CommonActions.java

```diff
@@ -22,24 +22,16 @@
 import org.apache.iceberg.Table;
 import org.apache.spark.sql.SparkSession;
 
-public class Actions {
+public abstract class CommonActions {
```

Comment:
Does it have to stay public?

Reply:
Package private should be fine, switching

Reply:
Ah oh no it's not :( because we have tests that use this that don't live in the "Actions" package this breaks some of the tests.

Reply:
Basically this needs to be accessible to tests in iceberg.spark as well as iceberg.actions